src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String CSV_DATA_DELIMITER = ",";
+    private static final String INCOME_INVOICE_STRING = "supply";
+    private static final String OUTCOME_INVOICE_STRING = "buy";
+    private static final String TOTAL_SUPPLY_STRING = "result";
+    private static final int OPERATION_ID = 0;
+    private static final int OPERATION_AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/212730772-c12e8364-0c38-4188-9ae2-ebc7afcd45f0.png)
create 3 private methods:
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```